Backend/dhall-configs/dev/rider-app.dhall

```diff
@@ -63,6 +65,37 @@ let InfoBIPConfig =
 
 let WebengageConfig = { url = "https://st.in.webengage.com" }
 
+let sampleKafkaConfig
```

Comment:
Doesn't match the configs defined inside issue
```
data EventStreamConfig = KafkaStream KafkaConfig | LogStream LogConfig

data EventStreamList = [EventStream]
data EventStream = EventStream {
    name :: Text,
    config :: EventStreamConfig
}

data EventStreamMapping = [EventStreamMap]

data EventStreamMap = EventStreamMap {
    streamName :: Text,
    eventTypes :: [Text]
}
```

Reply:
Also, isn't it better to store `EventStreamMap` inside DB instead of dhall config.

Reply:
The config structure was discussed and changed later.